cli/flox/src/commands/init/mod.rs

```diff
@@ -659,7 +658,7 @@ async fn get_default_package(flox: &Flox, package: &AttrPath) -> Result<Provided
         );
         let query = Query::new(
             package.to_string().as_ref(),
-            Features::parse()?.search_strategy,
+            flox.features.search_strategy.clone(),
```

Comment:
nonblocking: if we add a `Copy` derive to Search strategy we wouldnt need to clone even